definitions/vertical_scaling.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Vertical Scaling
+status: Completed
+category: concept
+---
+## Canary Deployment
+
+### What it is
+Vertical scaling refers to adding more resources (CPU/RAM/DISK) to your server (database or application server is still remains one) as on demand.
+
+Vertical Scaling is most commonly used in applications and products of middle-range as well as small and middle-sized companies. One of the most common examples of Virtual Scaling is to buy an expensive hardware and use it as a Virtual Machine hypervisor (VMWare ESX).
+
+Vertical Scaling usually means upgrade of server hardware. Some of the reasons to scale vertically includes increasing IOPS (Input / Ouput Operations), amplifying CPU/RAM capacity, as well as disk capacit
```

Comment:
Also not broadly accurate. Vertical scaling doesn't need to be done with physical hardware and can be just adding higher resource limits/requests to a kubernetes pod.